### DISCUSSION TITLE
下载提示内容仍处于加密状态

### DISCUSSION THREAD

**Discussion by Sleep312** [pjy612/SteamManifestCache#17] [Question / Answer]
这种情况是不是解锁dlc导致的 但我没有主动去解锁只用了sai入库
![屏幕截图 2023-11-22 132015](https://github.com/pjy612/SteamManifestCache/assets/149823327/a770f462-a78d-47e5-9d8e-1b687fc6ee1c)

**Comment by pjy612** [comment]
大概率是的，可能是因为游戏配置问题。不过战地似乎跨平台，入了估计也玩不了，推荐找学习版

**Comment by ZEERDEER** [comment]
[点击此处立即解决](https://store.steampowered.com/app/1238810/) 18￥

**Reply by YueHen14** [reply]
正确的

**Comment by pjy612** [comment]
一般出现这场景是密钥写入失败。
失败原因极大概率是 开着 steam 入库后，没及时重启steam。
导致 steam 重新覆盖了 密钥信息。
推荐 入库完毕后 即刻用tools重启steam，或者 干脆直接 关闭Steam后 再用工具入库。
